recipes/PyYAML-ft/conda_build_config.yaml

```
@@ -0,0 +1,26 @@
+channel_sources:
+  - conda-forge
+  - conda-forge/label/python_rc,conda-forge
+
+python:
+  - 3.13.* *_cp313
+  - 3.13.* *_cp313t
+
+numpy:
+  - 2
+  - 2
+
+python_min:
+  - 3.13
+
+is_python_min:
+  - true
+  - true
```

```suggestion
  - false
```

Could you try `is_python_min = false` here?

---

Done! And now only Linux fails. This is really odd.